src/main/java/javaClasses/homework_4/heghine_khachatryan/overloading/Calculations.java

```diff
@@ -0,0 +1,60 @@
+package javaClasses.homework_4.heghine_khachatryan.overloading;
+
+public class Calculations {
+
+    public int getMean(int... number) {
+        int count = 0;
+        int sum = 0;
+        for (int i : number) {
+            sum += i;
+            count++;
+        }
+        if (count != 0) {
```

Comment:
Very nice! It is very good to have checking for border cases, good job!

Reply:
thanks